### DISCUSSION TITLE
[Docs] Broken links, typo ? Say it here!

### DISCUSSION THREAD

**Discussion by Mara-Li** [ObsidianPublisher/obsidian-github-publisher#231] [General]
Hello!
To help me with organization, if you found a typo, an error or a dead links in the documentation, just write a comments here. I will repear this when I have the time.

**Comment by Mara-Li** [comment]
From #230:
- Looks like all the docs internal text-links are broken here: [obsidian-publisher.netlify.app/plugin](https://obsidian-publisher.netlify.app/plugin/)

**Comment by ooker777** [comment]
why not let people PR for you?

**Reply by Mara-Li** [reply]
They can ! But sometimes peoples don't know how to do it. So, if they don't know but found an error, they can, a last, leave a message here!
